extensions/vertx/deployment/pom.xml

```diff
@@ -30,7 +30,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
-
+	<dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-reactive-streams-operators</artifactId>
```

Comment:
@cescoffier is it normal that we only add the runtime dependency to the deployment artifact? I.e. we don't add the deployment one here AND we don't add the runtime dependency to runtime.

Just checking.

Reply:
it should be in both.